src/types/interface.types.ts

```diff
@@ -60,3 +60,18 @@ export interface DaoTableData {
 export interface UnknownObject {
 	[key: string]: string | number | boolean;
 }
+
+export interface AlgodTokenHeader {
```

Comment:
Could you import those types from algo-builder?

Reply:
I will check if we export it from there, will update